addons/test_mail/tests/test_mail_message.py

```diff
@@ -25,8 +26,78 @@ def setUpClass(cls):
         })
         cls.Message = cls.env['mail.message'].with_user(cls.user_employee)
 
+    def test_mail_message_values_body_base64_image(self):
+        msg = self.env['mail.message'].with_user(self.user_employee).create({
+            'body': 'taratata <img src="data:image/png;base64,iV/+OkI=" width="2"> <img src="data:image/png;base64,iV/+OkI=" width="2">',
+        })
+        self.assertEqual(len(msg.attachment_ids), 1)
+        self.assertEqual(
+            msg.body,
+            '<p>taratata <img src="/web/image/{attachment.id}?access_token={attachment.access_token}" alt="image0" width="2"> '
+            '<img src="/web/image/{attachment.id}?access_token={attachment.access_token}" alt="image0" width="2"></p>'.format(attachment=msg.attachment_ids[0])
+        )
+
     @mute_logger('odoo.models.unlink')
-    def test_mail_message_values_no_document_values(self):
+    @users('employee')
+    def test_mail_message_values_fromto_long_name(self):
+        """ Long headers may break in python if above 78 chars as folding is not
+        done correctly (see task-2602862). """
```

Comment:
can we include task id in docstring ? (I mean the code is public, and other people might not have access to the task)

Reply:
Indeed. Added instead a "see commit" to avoid repeating yet again the same issue :)